visualization/app/codeCharta/util/customConfigHelper.spec.ts

```diff
@@ -23,7 +23,7 @@ describe("CustomConfigHelper", () => {
 				id: "invalid-md5-checksum",
 				name: "stubbedConfig1",
 				mapSelectionMode: CustomConfigMapSelectionMode.MULTIPLE,
-				assignedMaps: ["test.cc.json"],
+				assignedMaps: new Map([["checksum", "test"]]),
```

Comment:
[Optional]
I liked the ending ".cc.json" as it showed that we store a map name as value
```suggestion
				assignedMaps: new Map([["checksum", "test.cc.json"]]),
```